client/src/api/api.js

```diff
@@ -269,6 +286,7 @@ export const downloadAudioFile = async (
   )
   try {
     const { uri } = await downloadResumable.downloadAsync()
+    await putFileURI(vocabId, uri, 'audio')
```

Comment:
Idea: what if we group the possible filetypes as an "enum", this would allow us to avoid hardcoding and centralize the filetypes and discern invalid file types that don't exist. 
Resource: https://masteringjs.io/tutorials/fundamentals/enum